### DISCUSSION TITLE
Noob question! - Texture & depth in spoke/blender

### DISCUSSION THREAD

**Discussion by Justine-Kat** [mozilla/hubs#3755] [Spoke Questions] [answered]
Hi there. I'm working on building a scene in spoke that already exists in Vectorworks. I've been using Blender primarily as a way to convert the file Vectorworks>Blender>Spoke, I'm very new at Blender. When it finally gets to spoke the room no longer has much depth at all and when I look at it in VR it's completely flat other than colours that were present in the Vectorworks file.

Is there any way I can use blender to add texture and depth that will show up in Spoke?

Thanks,
Justine

**Comment by CrimsonZA** [comment, marked as answer]
Sounds like your materials are being affected by the environment lighting , if you want flat colours to remain consistent or to not be effected by any lighting use the background node from the world materials and replace your main shader node on the object material

**Reply by j-conrad** [reply]
@CrimsonZA is correct. It seems that, according to discussions in the glTF exporter github repository, the Background node is not really necessary. For Hubs, we've been using this way for a while with no issues, but if this changes at some point, the documentation will get updated.

For clarity, here's a screenshot of how an unlit material is set up:

![image](https://user-images.githubusercontent.com/31968496/127236542-585ed6ee-1db6-4d1e-ba1f-eeb85bf48f0f.png)
